W-15959952 successfactors dropdown (#2604)
mulesoft/docs-connectors: sap-successfactors/3.1/modules/ROOT/pages/sap-successfactors-connector-reference.adoc

* <<Default>>
* <<NTLM>> |  |  |

> | Data Center's EndPoint URL a| String | Data center's endpoint URL. The new data centers might not appear in the dropdown menu, however you can still manually enter the value of the new data center. |  |x

Reviewer: ```suggestion
| Data Center's EndPoint URL a| String | Data center's endpoint URL. If the new data centers don't appear in the dropdown list, you can manually enter the value of the new data center. |  |x
```
